doc/community/extensions.rst

```diff
@@ -133,6 +133,7 @@ Python
 
 * :gh:`Wilfred/flycheck-pyflakes` adds a Python syntax checker using Pyflakes.
 * :gh:`msherry/flycheck-pycheckers` adds a checker for Python that can run multiple syntax checkers simultaneously (Pyflakes, PEP8, mypy2/3, etc.).
+* :gh:`chocoelho/flycheck-prospector` adds prospector checker for Python syntax.
```

Comment:
Could you add a link to `prospector`?  While you are here, you could fix the missing link for `Pyflakes` above (I think adding the underscore `Pyflakes_` should do it).

Reply:
np, just did it. I also took the liberty to do the same for Mypy, if that's not an issue...